### PR TITLE
#41-Style artist section

### PR DESCRIPTION
**Pull request by munozirianni1988**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

**Comment by munozirianni1988** [comment]
Artist_Section

File: client/src/components/ArtistSection.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
+.artist{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-around;
+  }
+  
+  .text h4{
```
Comment (mferryRV) [inline client/src/components/ArtistSection.css]: Can we make. these classes specific to the artist section?

File: client/src/pages/Home.js [hunk 2]
```diff
@@ -70,8 +71,10 @@ export function Home() {
 				<Link to="/about/this/site">About</Link>
 				<br></br>
 				<Link to="/RhythmsPanelTestingPage">RhythmsPanelTestingPage</Link>
-
-				<Link to="/ModalTestingPage">Modal Testing Page</Link>
+				<br></br>
```
Comment (mferryRV) [inline client/src/pages/Home.js]: Watch this on merge conflicts

**Review by mferryRV — APPROVED** [review]
Approved on the condition that you adjust class names to be section-specific

**Pull request merged**